### PR TITLE
Phalcon module see number of records

### PR DESCRIPTION
**Pull request by Zaszczyk**
I have added new method seeNumberOfRecords in Phalcon module, to check number of records in the table.

### CODE REVIEW

File: src/Codeception/Module/Phalcon.php [hunk 1]
```diff
@@ -379,6 +379,29 @@ public function seeRecord($model, $attributes = [])
...
+     * @param string $model Model name
+     * @param int $number int number of records
+     * @param array  $attributes Model attributes
+     * @part orm
+     */
+    public function seeNumberOfRecords($model, $number, $attributes = [])
+    {
+        $records = $this->findRecords($model, $attributes);
+        if ($records->count() != $number) {
+            $this->fail("Couldn't find $number records of $model with " . json_encode($attributes) .". Found: ".$records->count()." records.");
```
Comment (sergeyklay) [inline src/Codeception/Module/Phalcon.php]: Could you please do it more neat, using `sprintf` or something like that. I prefer avoid too long lines

Comment (Zaszczyk) [reply src/Codeception/Module/Phalcon.php]: Sure, done.

**Comment by Zaszczyk** [comment]
Could you merge it? :)

**Comment by sergeyklay** [comment]
cc @Naktibalda

**Pull request merged**